recipe/conda_build_config.yaml

```diff
@@ -6,5 +6,13 @@ folly_build_ext:
   - None
   - "jemalloc"  # [not win]
 fmt:
-  - "10"
   - "9"
+  - "9"
+  - "10"
+libevent:
+  - "2.1.10"
+  - "2.1.12"
+  - "2.1.12"
```

Comment:
According to [the specification of `zip_keys`](https://docs.conda.io/projects/conda-build/en/stable/resources/variants.html#coupling-keys), this must build for:
 - fmt 9 + libevent 2.1.10
 - fmt 9 + libevent 2.1.12
 - fmt 10 + libevent 2.1.12

but [the recipe has not and instead have built for](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=768985&view=results):
 - fmt 10 + libevent 2.1.10
 - fmt 10 + libevent 2.1.12
 - fmt 9 + libevent 2.1.12

Reply:
A work-around is proposed by https://github.com/conda-forge/folly-feedstock/pull/137/.

Reply:
The issue is tracked by https://github.com/conda/conda-build/issues/4987.